src/components/scatterplot/Scatterplot.js

```diff
@@ -1,5 +1,8 @@
+import React from 'react';
+
 import { SelectableScatterplotLayer } from '../../layers';
 import { cellLayerDefaultProps, DEFAULT_COLOR } from '../utils';
+import CellEmphasisSubscriber from '../CellEmphasisSubscriber';
```

Comment:
There are a few other ways:
- move the `renderCellEmphasis` function up to the parent `AbstractSelectableComponent` class, though I don't think this solves the problem of re-usability
- make the `CellEmphasis` components siblings of the plot components (rather than children) and define them at a higher level of the DOM tree, possibly in the api.js layout configuration. This would probably require associating individual tooltip components with individual plot components somehow. But it should allow plot components to be re-used without depending on the `CellEmphasis` components.